packages/truffle-slither-analyze/index.js

```diff
@@ -0,0 +1,17 @@
+const which = require("which");
+const { spawn } = require("child_process");
+
+function truffleSlitherAnalyze(options) {
+  try {
+    let slither = which.sync("slither");
+    let child = spawn(slither, options);
+    child.stdout.on("data", data => console.log(data.toString()));
+    child.stderr.on("data", data => console.log(data.toString()));
+  } catch (error) {
+    console.log("Slither not found in path");
+    console.log(`Please install the package using \`pip install slither\`\n`);
```

Comment:
```suggestion
    console.log(`Please install the package using \`pip install slither-analyzer\`\n`);
```

Reply:
Oh, sorry about that. I had built it from source, never actually installed from pypi.